docs/content/using/csi_ug_using_sample.md

```diff
@@ -20,20 +20,20 @@ The following are examples of different types of secret names that can be given
 
 Use this information to run a stateful container on StatefulSet volumes using either file systems or raw block volumes.
 
-1.  Create an array secret, as described in [Creating a Secret](../configuration/csi_ug_config_create_secret.md).
+1. Create an array secret, as described in [Creating a Secret](../configuration/csi_ug_config_create_secret.md).
 
-2.  Create a storage class, as described in [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
+2. Create a storage class, as described in [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
 
     **Remember:** The `SpaceEfficiency` values for Spectrum Virtualize Family are: thick, thin, compressed, or deduplicated. These values are not case specific.
     
     For DS8000 Family systems, the default value is standard, but can be set to thin, if required. These values are not case specific. For more information, see [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
```

Comment:
@ArbelNathan is `standard` == `none` in ds8k-to-CSI terminology?

Reply:
no, default is `none`, there is no `standard`

Reply:
I'll update to `none` but this was written as part of CSI-1234 on March 9.

Reply:
I have no idea where Nicole got that value in CSI-1234

Reply:
DONE